liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html

```diff
@@ -9,7 +9,8 @@
                 {% image page.image width-770 as image %}
                 <img src="{{ image.url }}" alt="{{ image.alt }}" class="homepage-header__image" />
                 <div class="homepage-header__text">
-                    <h1 class="cms-title">{{ page.title }}</h1>
+                    <h1 class="homepage-title homepage-title--large">Beteiligung.in/<span class="marked cms-title--normal">jederstadt</span></h1>
+                    <h1 class="homepage-title homepage-title--small">Beteiligung.in/<br><span class="marked cms-title--normal">jederstadt</span></h1>
```

Comment:
would it be possible to use a pseudo `:after` selector to add the break on small screens?